### ISSUE TITLE
HTTP 400 error in loading app in TrueNas Scale

### ISSUE THREAD

**Issue by Strobeener** [qdm12/ddns-updater#702]
Getting HTTP 400 error.   It is seeing my account and finding my ip address but stalls out.

I attached the log, but in case it doesn't attach here is the last few  line s

```
2024-04-29 03:47:20.338217-07:002024-04-29T06:47:20-04:00 ERROR [backup] open /updater/data/updates.json: no such file or directory
2024-04-29 03:47:20.451832-07:002024-04-29T06:47:20-04:00 INFO ipv4 address of woodland-geeks.com is 104.21.32.230 and your ipv4 address  is 24.247.108.2
2024-04-29 03:47:20.451866-07:002024-04-29T06:47:20-04:00 INFO Updating record [domain: woodland-geeks.com | host: @ | provider: cloudflare | ip: ipv4] to use 24.247.108.2
2024-04-29 03:47:20.574438-07:002024-04-29T06:47:20-04:00 ERROR getting record id: HTTP status is not valid: 400: {"success":false,"errors":[{"code":9106,"message":"Missing X-Auth-Key, X-Auth-Email or Authorization headers"}]}
```

**TLDR**: *Describe your issue in a one liner here*

1. Is this urgent: No
2. DNS provider(s) you use: Cloudflare
3. Program version:

    Running version 2.6.0 built on 2024-02-03T18:53:40.722Z (commit c338c28)

    

4. What are you using to run the container: TrueNas Chart
5. Extra information (optional)

```json
{
  "settings": [
    {
      "api_key": "api-key",
      "domain": "domain name  note my domain name is an A record with just domain.com",
      "email": "email",
      "host": "@",
      "ip_version": "ipv4",
      "provider": "cloudflare",
      "proxied": false,
      "ttl": 1,
      "zone_identifier": "zoneidentfier"
    }
  ]
}
```

Host OS:  TrueNAS-SCALE-23.10.2
[ddns-updater_ddns-updater-6fb7bdb97f-tt4bs_ddns-updater.log](https://github.com/qdm12/ddns-updater/files/15148161/ddns-updater_ddns-updater-6fb7bdb97f-tt4bs_ddns-updater.log)

**Comment by qdm12** [comment]
Please run it with `LOG_LEVEL=debug` to see http requests and responses and report back the log. Make sure to remove your credentials from the logs.

**Comment by Strobeener** [comment]
This is going to sound very newbie cause quite frankly I am trying to learn this stuff.  But does LOG_LEVEL=debug  go in as an environment variable?


Sent from Yahoo Mail for iPhone


On Tuesday, April 30, 2024, 5:23 AM, Quentin McGaw ***@***.***> wrote:




Please run it with LOG_LEVEL=debug to see http requests and responses and report back the log. Make sure to remove your credentials from the logs.

**Comment by qdm12** [comment]
Yes :wink:

**Comment by Strobeener** [comment]
hmmmm... you can close the issue thanks for helping.   I was out of town for work yesterday.  I re-installed the app (after I deleted the files from the dataset ---stored on host).  I went through setup and used my cloudflare token instead of Global API Key and email and everything is working :)
Not sure why - but as i am trying to learn all this stuff , i follow youtube videos or others walkthroughs and it never works for me at first  Takes forever of trying and trying and then poof it works
    On Tuesday, April 30, 2024 at 07:47:45 AM EDT, Quentin McGaw ***@***.***> wrote:  
 
 


Yes 😉

**Comment by qdm12** [comment]
Nice, thanks for letting me know.
